### ISSUE TITLE
Please tag a new release (1.1?)

### ISSUE THREAD

**Issue by filbranden** [jhawthorn/fzy#147]
Latest tag 1.0 is from almost 2 years ago (September 2018) and there have been 45 commits since that release.

Tagging a new one would make it easier on Linux distributions to state they carry version 1.1 (rather than 1.0 + a long set of commits since...)

Thanks!
Filipe

**Comment by hmh** [comment]
Indeed. Debian and Ubuntu are still distributing 1.0, likely because a new release never gets tagged...

**Comment by cglogic** [comment]
FreeBSD is also stuck on 1.0, please make a new release.
